### ISSUE TITLE
InceptionV3 normalisation

### ISSUE THREAD

**Issue by gurkirt** [Cadene/pretrained-models.pytorch#7]
Hi-great work with putting all the models together. 

Model mean and std given at https://github.com/Cadene/pretrained-models.pytorch#modelmean for inceptionv3 are different from what is provided with pytorchVison https://github.com/pytorch/vision#models. Where they say "The images have to be loaded in to a range of [0, 1] and then normalized using mean=[0.485, 0.456, 0.406] and std=[0.229, 0.224, 0.225]" for all the models including inception v3. 

Am I missing something?

**Comment by Cadene** [comment]
InceptionV3 from both lib have the same arch and parameters (ported from tensorflow).

The only difference is that torchvision apply a transformation to keep the mean and std the same for all the pretrained networks.
https://github.com/pytorch/vision/blob/master/torchvision/models/inception.py#L72-L76

I prefer to keep the original mean and std, but that is just a matter of taste.

**Comment by gurkirt** [comment]
I think your way is better one. Transform is applied only once, so less computation.
